synfig-studio/ui/shortcuts_window.glade

```diff
@@ -87,6 +87,33 @@
             </child>
           </object>
         </child>
+        <child>
+          <object class="GtkShortcutsGroup">
+            <property name="visible">true</property>
+            <property name="title" translatable="yes" context="shortcuts window">Canvas Zoom</property>
+            <child>
+              <object class="GtkShortcutsShortcut">
+                <property name="visible">true</property>
+                <property name="title" translatable="yes" context="shortcuts window">Canvas-Zoom-In</property>
+                <property name="accelerator">&lt;Ctrl&gt;equal</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkShortcutsShortcut">
+                <property name="visible">true</property>
+                <property name="title" translatable="yes" context="shortcuts window">Canvas-Zoom-Out</property>
+                <property name="accelerator">&lt;Ctrl&gt;minus</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkShortcutsShortcut">
+                <property name="visible">true</property>
+                <property name="title" translatable="yes" context="shortcuts window">Canvas-Zoom-Fit</property>
+                <property name="accelerator">&lt;Ctrl&gt;0</property>
+              </object>
+            </child>			
+          </object>
+        </child>		
```

Comment:
You used the existing section ie., **Navigation Shortcuts**

I think it's better moving those shortcut groups to a new section called **Zoom Shortcuts**

![Screenshot from 2020-11-15 20-07-04](https://user-images.githubusercontent.com/24214509/99187998-c850cb00-277f-11eb-80be-f5d2b9165c71.png)

Reply:
Also, there are two alternative shortcuts for the same given action:
- Zoom in &nbsp;&nbsp;: `Ctrl + =` and `equal`
- Zoom out : `Ctrl + -` and `minus`
- Zoom fit &nbsp;&nbsp;: `Ctrl + 0` and `0`

We need to mention both instead of one.

Here's your reference doc [GtkShortcutsShortcut](https://developer.gnome.org/gtk3/stable/GtkShortcutsShortcut.html#GtkShortcutsShortcut--accelerator). The answer's in the examples :)

Reply:
Got it . I am starting working on this. Thanks for review and tips :)

Reply:
Ping. You missed to update the alternative shortcuts in you new commit.

Reply:
@Keyikedalube Ohh...sorry I missed them. I will also add them in the shortcuts option. BTW I was getting confused by the image you linked above. Where is this menu in synfig ?? I am unable to find it. Maybe its in newer build coz i am using a 1 month old build or I m just unable to find it. Kindly tell me.
Thanks :)

Reply:
No, there's no menu.

You only have to place the objects properly in .glade file. Gtk will take care of the menu automatically :)

Reply:
@Keyikedalube Is there any way to just build a particular thing ( just the shortcuts box for example ) so that i can test my changes. Thanks :)

Reply:
For synfig we're not building glade file because we're using `create/add_from_file()` API. You only have to modify the glade file and copy that to build output and then run synfigstudio to see the changes :)

In other words no need to call CMake or Autotools script for now.

Reply:
Resolved :)

BTW, can you click on those buttons "Resolve Conversation" for the ones I've already mentioned that word? Over here I can't. This privilege is given to author LOL

Reply:
Ohh Yes !! shortcuts window worked without building !! Thank You !! now i guess those files which are being use as it is in the build can be simply modified by just replacing them !! I will find out more about this in future , after all it is saving me from building. ;)